packet/baseapi.py

```diff
@@ -47,12 +49,21 @@ class BaseAPI(object):
         Basic api class for
     """
 
-    def __init__(self, auth_token, consumer_token):
+    def __init__(self, auth_token, consumer_token, user_agent=None):
         self.auth_token = auth_token
         self.consumer_token = consumer_token
         self.end_point = "api.packet.net"
+        self._user_agent_prefix = user_agent
         self._log = logging.getLogger(__name__)
 
+    @property
+    def user_agent(self):
+        return "{}packet-python/{} {}".format(
+            "{} ".format(self._user_agent_prefix) if self._user_agent_prefix else "",
+            __version__,
+            requests.utils.default_user_agent(),
+        )
```

Comment:
If you go with `user_agent=""` suggestion above this could instead be:
```python
        return " ".join((self._user_agent_prefix, "packet-python/"+__version__, requests.utils.default_user_agent()).strip()
```

or

```python
        return "{} packet-python/{} {}".format(
            self._user_agent_prefix, 
            __version__,
            requests.utils.default_user_agent()).strip()
```

Reply:
Thanks, @mmlb!

How's it looking now, https://github.com/packethost/packet-python/compare/160ef0653d4bb35700d2bc45930ffee1b33b8b73..7a55c8fb836546a703fc42a47697266e0816e773?